Lib/http/cookies.py

```diff
@@ -273,18 +273,19 @@ class Morsel(dict):
     # variant on the left to the appropriate traditional
     # formatting on the right.
     _reserved = {
-        "expires"  : "expires",
-        "path"     : "Path",
-        "comment"  : "Comment",
-        "domain"   : "Domain",
-        "max-age"  : "Max-Age",
-        "secure"   : "Secure",
-        "httponly" : "HttpOnly",
-        "version"  : "Version",
-        "samesite" : "SameSite",
+        "expires": "expires",
+        "path": "Path",
+        "comment": "Comment",
+        "domain": "Domain",
+        "max-age": "Max-Age",
+        "secure": "Secure",
+        "httponly": "HttpOnly",
+        "version": "Version",
+        "samesite": "SameSite",
```

Comment:
Let’s not touch existing lines (better for git history etc)

Reply:
We can't avoid it either way, because if we keep the spacing alignment we'll need to realign all the lines to `partitioned`, being the longest key now. 

I'm going to retain @gpshead's preference that we drop the alignment spacing in my rebase commit, and happy to follow whatever consensus emerges if you disagree.

Reply:
No, keep the existing lines as is and don’t follow the alignment for the new line. Easiest thing to do!